curriculum/challenges/english/25-front-end-development/workshop-node-and-express/587d7fb0367417b2b2512bed.md

```diff
@@ -18,15 +18,15 @@ During the development process, it is important to be able to check what’s goi
 
 Node is just a JavaScript environment. Like client side JavaScript, you can use the console to display useful debug information. On your local machine, you would see console output in a terminal. On Gitpod, a terminal is open at the bottom of the editor by default.
 
-We recommend to keep the terminal open while working at these challenges. By reading the output in the terminal, you can see any errors that may occur.
+It is recommended to keep the terminal open while working at these challenges. By reading the output in the terminal, you can see any errors that may occur.
 
 The server must be restarted after making changes to its files.
 
 You can stop the server from the terminal using `Ctrl + C` and start it using Node directly (`node mainEntryFile.js`) or using a run script in the `package.json` file with `npm run`.
 
 For example, the `"start": "node server.js"` script would be run from the terminal using `npm run start`.
 
-To implement server auto restarting on file save Node provides the `--watch` flag you can add to your start script `"start": "node --watch server.js"` or you can install an npm package like `nodemon`. We will leave this to you as an exercise.
+To implement server auto restarting on file save Node provides the `--watch` flag you can add to your start script `"start": "node --watch server.js"` or you can install a NPM package like `nodemon`. This will be left to you as an exercise.
```

Comment:
```suggestion
To implement server auto restarting on file save Node provides the `--watch` flag you can add to your start script `"start": "node --watch server.js"` or you can install a npm package like `nodemon`. This will be left to you as an exercise.
```